common/workerpool.go

```diff
@@ -0,0 +1,14 @@
+package common
+
+import "context"
+
+type WorkerPool interface {
```

Comment:
shall we comment this is tied to gammazero workerpool, instead of trying to be generic to support different impls? There are others like ants which have difference interfaces.